### ISSUE TITLE
Microscope parameters saved in calibration_metadata may be inaccurate

### ISSUE THREAD

**Issue by ieivanov** [mehta-lab/recOrder#216]
Recorder populates default microscope parameters in the Acquisition/Reconstruction tab, and saves these parameters in the calibration_metadata file even if the user has not modified the default values. This can result in inaccurate metadata, especially since these params are in a different tabs, and the user is not prompted to enter them. 

Here we're trying to balance convenience and capturing all metadata. One good option may be to leave these fields blank and add a "Populate with default values" button, which can simplify our lives a bit.

**Comment by talonchandler** [comment]
@ieivanov and I just agreed that we can drop the "Microscope parameters" fields from `calibration_metadata.txt`. This will prevent default values that the user has not seen from showing up in our metadata files, and it will allow us to side-step some of the GUI design issues that @mattersoflight and I discussed today. 

@ieivanov and I also discussed how default values will be treated in 1.0.0. We roughly settled on  a combination of 
- sensible defaults for first startup, 
- auto-load from cached parameters for subsequent startups (it's annoying to re-enter at startup), and
- save/load from file 
features will handle all of our use cases. 

Feel free to add or comment @ieivanov.
